charts/nr-k8s-otel-collector/templates/deployment-configmap.yaml

```diff
@@ -0,0 +1,334 @@
+
+      resource/metrics:
+        attributes:
+          - key: host.id
+            from_attribute: host.name
+            action: upsert
+          # TODO chris: Upsert only when cluster name not found (resource detection override: true)
+          - key: k8s.cluster.name
+            action: upsert
+            value: {{ include "newrelic.common.cluster" . }}
+          - key: newrelicOnly
+            action: upsert
+            value: 'true'
+          - key: service.name
+            action: delete
+          - key: service_name
+            action: delete
+
+      batch:
+      
+      resource/events:
+        attributes:
+          - key: "event.name"
+            action: upsert
+            value: "InfrastructureEvent"
+          - key: "event.domain"
+            action: upsert
+            value: "newrelic-otel-event"
+          - key: "category"
+            action: upsert
+            value: "kubernetes"
+          - key: k8s.cluster.name
+            action: upsert
+            value: {{ include "newrelic.common.cluster" . }}
+          - key: newrelicOnly
+            action: upsert
+            value: 'true'
+
+      k8sattributes:
+        auth_type: "serviceAccount"
+        passthrough: false
+        filter:
+          node_from_env_var: KUBE_NODE_NAME
+        extract:
+          metadata:
+            - k8s.pod.name
+            - k8s.pod.uid
+            - k8s.deployment.name
+            - k8s.namespace.name
+            - k8s.node.name
+            - k8s.pod.start_time
+        pod_association:
+          - sources: 
+            - from: resource_attribute
+              name: k8s.pod.uid
+
+    exporters:
+      otlp:
+        endpoint: "nr-k8s-otel-collector-gateway:4317"
+        tls:
+          insecure: true
+      otlphttp/newrelic:
+        endpoint: {{ .Values.newRelicEndpoint }}
+        headers:
+          api-key: ${env:NR_LICENSE_KEY}
+      debug:
+        verbosity: detailed
+        sampling_initial: 5
+        sampling_thereafter: 200
+    service:
+      telemetry:
+        logs:
+          level: "debug"
+      pipelines:
+        # Uncomment below if exporting traces to NR
+        # traces:
+        #   receivers: [otlp]
+        #   processors: [batch]
+        #   exporters: [otlphttp/newrelic, debug]
+        metrics:
+          receivers: [otlp, prometheus, k8s_cluster]
+          processors: 
+            - metricstransform/k8s_cluster_info
+            - metricstransform/kube_pod_status_phase
+            - filter/exclude_zero_value_kube_pod_container_status
+            - resource/metrics
+            - batch
+            - k8sattributes
+          exporters: [otlphttp/newrelic, debug]
+        logs:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [otlphttp/newrelic, debug]
+        logs/events:
+          receivers: [k8s_events]
+          processors: [resource/events, batch]
+          exporters: [otlp, debug]
```

Comment:
I have the feeling we are sending data to the same component sending the data 😅

Reply:
Hmmm, let me circle back to you on this

Reply:
updated